CS/CASESTUDY1/Vehicle.java

```diff
@@ -0,0 +1,82 @@
+
+public class Vehicle {
```

Comment:
Sir USA sir told that all common attributes's Getter Setter method will be in vehicle class and only calculateOnRoadPrice() method will be overwritten by its child class.